randovania/gui/multiworld_client.py

```diff
@@ -128,15 +134,26 @@ async def _server_sync(self):
 
             try:
                 result = await self.network_client.perform_world_sync(request)
-            except (Exception, UnableToConnect) as e:
-                message = f"Exception {type(e)} when attempting to sync locations."
-                if isinstance(e, (UnableToConnect, error.NotLoggedIn, error.InvalidSession, error.RequestTimeout)):
-                    self.logger.warning(message)
-                else:
-                    self.logger.exception(message)
-                await asyncio.sleep(5)
+
+            except (UnableToConnect, error.RequestTimeoutError) as e:
```

Comment:
is it within scope to rename this to `UnableToConnectError`?

Reply:
I only renamed stuff that subclass BaseNetworkError.

Eventually I'll do the rest, it's a rule you can enable in ruff